types/yaireo__tagify/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/yaireo__tagify",
-    "version": "4.18.9999",
+    "version": "4.24.9999",
```

Comment:
4.24 or 4.23? The PR title and description seem to indicate 4.23.

Reply:
While I was waiting for a review, a new version has been released including a little update. That's why I updated this pull request (didn't change the title though). In the meantime, there are actually two new minor releases.